packages/plugins/ansible-foreman_scap_client/ansible-foreman_scap_client.spec

```diff
@@ -0,0 +1,40 @@
+%global repo_orgname theforeman
+%global repo_name ansible-foreman_scap_client
+
+%global role_orgname %{repo_orgname}
+%global role_name foreman_scap_client
+
+Name: ansible-foreman_scap_client
```

Comment:
For the [insights client](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec) we're using the `ansiblerole-` prefix instead of `ansible-`.

While there still isn't an official packaging guideline in Fedora how to name Ansible roles, I think we should be at least consistent in our own repo and call this one `ansiblerole-foreman_scap_client`.

(see https://github.com/theforeman/foreman-packaging/pull/2286#issuecomment-383150168 for the discussion back then with the insights client package)